wandb/sdk/launch/_project_spec.py

```diff
@@ -54,8 +61,20 @@ def __init__(
         if "entry_point" in overrides:
             self.add_entry_point(overrides["entry_point"])
 
+        if utils._is_wandb_uri(self.uri):
+            self.source = LaunchSource.WANDB
+            self.project_dir = tempfile.mkdtemp()
+        elif utils._is_github_uri(self.uri):
+            self.source = LaunchSource.GITHUB
+            self.project_dir = tempfile.mkdtemp()
+        else:
+            # assume local
+            self.source = LaunchSource.LOCAL
+            self.project_dir = self.uri
+
+        self.aux_dir = tempfile.mkdtemp()
+
         self.run_id = generate_id()
-        self.dir = tempfile.mkdtemp()
         self.config_path = DEFAULT_CONFIG_PATH
```

Comment:
I think the config path should be `os.path.join(self.aux_dir, DEFAULT_CONFIG_PATH)`

Reply:
oh yeah it does this in the copy fn but looks like the docker side assumes `config_path` is the full, good catch